packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -73,7 +73,7 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
     message: 'Add questions',
   },
   selectFoldersOrExercises: {
```

Comment:
Might be worth changing the message id if we're not only allowing selection of resources?

Reply:
+1 good thought richard